ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_common.py

```diff
@@ -186,3 +186,59 @@ def test_paddle_zeropad2d(
         padding=padding,
         data_format=dataformat,
     )
+
+
+# linear
+@st.composite
+def x_and_linear(draw, dtypes):
+    dtype = draw(dtypes)
+    in_features = draw(helpers.ints(min_value=1, max_value=1))
+    out_features = draw(helpers.ints(min_value=1, max_value=1))
+
+    x_shape = (
+        1,
+        1,
+        in_features,
+    )
+    weight_shape = (out_features, in_features)
+    bias_shape = (out_features,)
+
+    x = draw(
+        helpers.array_values(dtype=dtype[0], shape=x_shape, min_value=0, max_value=1)
+    )
+    weight = draw(
+        helpers.array_values(
+            dtype=dtype[0], shape=weight_shape, min_value=0, max_value=1
+        )
+    )
+    bias = draw(
+        helpers.array_values(dtype=dtype[0], shape=bias_shape, min_value=0, max_value=1)
+    )
+    return dtype, x, weight, bias
+
+
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.linear",
+    dtype_x_weight_bias=x_and_linear(
+        dtypes=helpers.get_dtypes("float", full=False),
```

Comment:
Also, the best practice is to use "valid" here instead of "float". The testing pipeline automatically samples the supported dtypes based on the function's decorator.